packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -37,7 +37,20 @@ describe('request-queue tests', () => {
     chai.assert.isObject(queue._config);
   });
 
-  it('queueName is corrent', () =>{
+  it('initialize should re-fill the queue', async () => {
```

Comment:
I may be misunderstanding the original problem, but this test doesn't actually test what happens when you call initQueue when there are no entries to be added.